apps/rpc-server/docker_files/entrypoint.sh

```diff
@@ -39,7 +39,7 @@ fi
 
 ### Store container env vars for rpc-server
 cat <<EOF > /docker.env
-CLIENT=$CLIENT
+CLIENT=${CLIENT}
```

Comment:
I am ok with this failing during docker build because RPC server will fail without it.
It is the only mandatory env var.